src/audio/dai-zephyr.c

```diff
@@ -639,6 +639,11 @@ static int dai_playback_params(struct dai_data *dd, struct comp_dev *dev, uint32
 			goto out;
 		}
 
+		if (!max_block_count) {
```

Comment:
better to combine this with line 637 with:
if (err < 0 || !max_block_count) {

since they have same cause from dma_get_attribute

Reply:
two different conditions though, deserve separate handling